core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataMappingService.java

```diff
@@ -114,50 +89,30 @@ public MetaDataMappingService(Settings settings, ClusterService clusterService,
      * as possible so we won't create the same index all the time for example for the updates on the same mapping
```

Comment:
just above this line, update operations can go away - it's refresh only these days.